model/src/logger.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
nice! 😄

no need to action, maybe Logger should be a service (DI) instead of init-ed by the object using it, and persistenceService.log should call the service and add the name itself, instead of logger

Reply:
Perhaps. I'm not 100% sure how we'd go about implementing the DI (Haven't really worked out how that works yet!). I really wanted to avoid the log methods we had before (I think what you're suggesting in your second comment?), as they feel far too tied into the existing logger implementation. This felt like a happy compromise (create the logger with a name and then just use it over and over). Perhaps our DS service could really be a factory for loggers - so our loggerFactory gets injected, and we ask it for a logger with a given name (avoiding any knowledge of how the logger is instantiated in our client classes)

Reply:
we have DI implemented elsewhere using `schmervice`. you'll have come across persistenceService and cacheService which are all DI'd

Reply:
Cool, will leave as is for now, perhaps we can address later in a spare moment!